web/app.js

```diff
@@ -1655,7 +1655,8 @@ const PDFViewerApplication = {
           baseURL: this.baseUrl,
           filesize: this._contentLength,
           filename: this._docFilename,
-          metadata: this.metadata,
+          metadata: this.rawMetadata,
+          authors: this.metadata.get("authors") || null,
```

Comment:
Note that `this.metadata` can be null, hence this can throw!
Furthermore, the `get` method already fallback to returning `null`.

All in all, this line should read
```suggestion
          authors: this.metadata?.get("authors"),
```